packages/api/server.js

```diff
@@ -20,7 +20,9 @@ const bodyParser = require('body-parser');
 const PostBuildHandler = require('./src/post-build.js');
 const { Firestore } = require('@google-cloud/firestore');
 const client = new Firestore();
+const cors = require('cors');
 
+app.use(cors());
```

Comment:
@cedpeters @StuartRucker this here just needs to be added to allow the browser to talk to the backend API.